python/sglang/srt/layers/dp_attention.py

```diff
@@ -261,3 +261,36 @@ def tp_reduce_scatter(
 
 def tp_all_gather(output_list: List[torch.Tensor], input_: torch.Tensor):
     return get_attention_tp_group().all_gather(input_, tensor_list=output_list)
+
+
+def dp_reduce_scatter(
```

Comment:
This function performs reduce-scatter across the global TP group. We may call it `tensor_model_parallel_reduce_scatter` and place it to a different file.

Reply:
I think just `tp_reduce_scatter` is sufficient